cosmos/models/Task.py

```diff
@@ -172,6 +172,7 @@ class Task(Base):
     attempt = Column(Integer, nullable=False)
     must_succeed = Column(Boolean, nullable=False)
     drm = Column(String(255))
+    job_class = Column(String(255))
```

Comment:
@egafni you'd mentioned in another comment that this PR looks good to you and "doesn't break the API or require a DB migration so that's cool." -- In that light, I wanted to bring this line (an additional field on Task) to your attention -- will this cause problems for other folks upgrading from previous versions?

Reply:
Yeah - I don't think this is important enough to break people's DBs

Reply:
I think just comment this line and you're good?

Reply:
Apologies in advance for my lack of understanding. Seems to me that there'll increasingly be DRM-specific options we'll want to tack on to a task. Does it make sense to establish something like a key-value store rather than having the Task itself contain the union of options for every DRM?

Reply:
Josh, that's a great question that I mentioned obliquely in a comment around here somewhere... I am happy adding a single, named field for this use case, as I don't anticipate adding any others in the short- to medium-term. Changing Task to have a dict instead of a named field for queue would be a breaking change for other Cosmos users. (Erik's indicated he'd like to avoid that which is fine by me.) It probably is something to think about in the longer-term, or to consider adding the next time Erik plans to make model changes.

@egafni what do you think?

Reply:
Adding this field breaks old DBs for people.  I don't think *storing* job_class in the database is important enough to warrant that backwards incompatibility (correct me if you think it's really important - it is probably stored in sacct though).

I think just storing the attribute in memory on the Task instance for the lifecycle of the process is sufficient?